test/Base_Tests/src/Data/Numbers_Spec.enso

```diff
@@ -101,6 +101,9 @@ add_specs suite_builder =
             (3 / 4) . should_equal 0.75 epsilon=eps
             (almost_max_long * 2 / almost_max_long_times_three) . should_equal 0.6666666 epsilon=eps
 
+            1/0 . is_infinite . should_be_true
+            -1/0 . is_infinite . should_be_true
```

Comment:
not a problem, but out of curiosity - why is this test moved?

Reply:
Because it was originally with the modulo tests, and I thought it should be with the floating point division tests.

Reply:
Ah, makes sense